### PR TITLE
KAFKA-10587 Rename kafka-mirror-maker CLI command line arguments for KIP-629

### PR DESCRIPTION
**Pull request by OmniaGM**
[KAFKA-10587](https://issues.apache.org/jira/browse/KAFKA-10587) Rename kafka-mirror-maker CLI command line arguments for KIP-629
Replace "whitelist" argument in kafka-mirror-maker cli command with "include"

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by OmniaGM** [comment]
Hi, @xvrl, @mimaison can you please have a look into this? Thanks

**Review by showuon — COMMENTED** [review]
@OmniaGM , thanks for the PR. Overall LGTM. Could you please also update the test file for MirrorMaker? (i.e. `MirrorMakerIntegrationTest.scala`) Thanks.

**Comment by OmniaGM** [comment]
> Overall LGTM. Could you please also update the test file for MirrorMaker? (i.e. MirrorMakerIntegrationTest.scala) Thanks.

@showuon Thanks for having a look, the pr already includes `MirrorMakerIntegrationTest.scala`. I did a quick search and from the look of it, this is the only MM2 test file that mentioned racial terms.

**Review by showuon — APPROVED** [review]
Sorry, I missed that file. LGTM! Thank you.

File: core/src/main/scala/kafka/tools/MirrorMaker.scala [hunk 1]
```diff
@@ -454,8 +454,8 @@ object MirrorMaker extends Logging with KafkaMetricsGroup {
       .ofType(classOf[java.lang.Integer])
       .defaultsTo(1)
 
-    val whitelistOpt = parser.accepts("whitelist",
-      "Whitelist of topics to mirror.")
+    val includeOpt = parser.accepts("include",
```
Comment (xvrl) [inline core/src/main/scala/kafka/tools/MirrorMaker.scala:463]: we should deprecate the old argument, but provide backwards compatibility until we remove it in a future release.

Comment (OmniaGM) [reply core/src/main/scala/kafka/tools/MirrorMaker.scala:463]: I added it back and handled both. Now the command will have 
```
--whitelist <String: Java regex          DEPRECATED, use --include instead;    
  (String)>                                ignored if --include specified. List
                                           of included topics to mirror. 
--include <String: Java regex (String)>  List of included topics to mirror.    
```

File: core/src/main/scala/kafka/tools/MirrorMaker.scala [hunk 2]
```diff
@@ -455,7 +455,13 @@ object MirrorMaker extends Logging with KafkaMetricsGroup {
       .defaultsTo(1)
 
     val whitelistOpt = parser.accepts("whitelist",
-      "Whitelist of topics to mirror.")
+      "DEPRECATED, use --include instead; ignored if --include specified. List of included topics to mirror.")
```
Comment (xvrl) [inline core/src/main/scala/kafka/tools/MirrorMaker.scala:458]: nit, the term "included" seems redundant
```suggestion
      "DEPRECATED, use --include instead; ignored if --include specified. List of topics to mirror.")
```

File: core/src/main/scala/kafka/tools/MirrorMaker.scala [hunk 3]
```diff
@@ -455,7 +455,13 @@ object MirrorMaker extends Logging with KafkaMetricsGroup {
...
+      .withRequiredArg()
+      .describedAs("Java regex (String)")
+      .ofType(classOf[String])
+
+    val includeOpt = parser.accepts("include",
+      "List of included topics to mirror.")
```
Comment (xvrl) [inline core/src/main/scala/kafka/tools/MirrorMaker.scala:464]: same as above
```suggestion
      "List of topics to mirror.")
```

**Review by xvrl — APPROVED** [review]
thanks @OmniaGM!
LGTM, just a few minor nits. We'll also need a Kafka committer to approve and get it merged.

**Review by gwenshap — APPROVED** [review]
LGTM

**Pull request closed without merging**